polling_stations/apps/pollingstations/context_processors.py

```diff
@@ -10,7 +12,7 @@ def google_analytics(request):
     settings, "NEXT_CHARISMATIC_ELECTION_DATE", None
 )
 NEXT_CHARISMATIC_ELECTION_DATE = (
-    datetime.strptime(NEXT_CHARISMATIC_ELECTION_DATE, "%Y-%m-%d")
+    datetime.strptime(NEXT_CHARISMATIC_ELECTION_DATE, "%Y-%m-%d").date()
```

Comment:
Will this affect usage in templates: https://github.com/DemocracyClub/UK-Polling-Stations/blob/eaaa1a23eadb0cb1844222fbacbc099b7bf869a6/polling_stations/templates/base_full.html#L13

Reply:
If anything it'll make it better. For some reason I couldn't get Django to parse the date using `|date` when it was a `datetime` object. Tested locally and that line is working.